source/defender.c

```diff
@@ -205,12 +205,13 @@ static uint32_t getTopicLength( uint16_t thingNameLength,
             topicLength = DEFENDER_API_CBOR_REPORT_ACCEPTED_LENGTH( thingNameLength );
             break;
 
+        /* The default is here just to silence compiler warnings in a way which
+         * does not bring coverage down. The assert at the beginning of this
+         * function ensures that the only api value hitting this case can be
+         * DefenderCborReportRejected. */
         case DefenderCborReportRejected:
-            topicLength = DEFENDER_API_CBOR_REPORT_REJECTED_LENGTH( thingNameLength );
-            break;
-
         default:
-            /* Invalid. */
+            topicLength = DEFENDER_API_CBOR_REPORT_REJECTED_LENGTH( thingNameLength );
```

Comment:
I'm assuming this kind of thing is unrelated to the cbmc proofs.

Reply:
You are right and this is not related to cbmc proofs.